deployment/manage_autocompletion.sh

```diff
@@ -20,25 +20,4 @@ _managepy_complete()
     fi
 }
 
-_python_complete()
```

Comment:
From my commit message:

> It turns out that nix shells have a nice way to load completions
> already. This even uses "bash" in the installation, so that it does not
> interfere with other shells that may load the nix environment with
> direnv.
> 
>     https://togithub.com/NixOS/nixpkgs/pull/103501
>     https://togithub.com/scop/bash-completion/blob/7f1165018a72f837d59b3cea7071efa0108841a2/README.md
> 
> Since I now used the "lazy" approach for loading the completions, the
> completion on `python manage.py <Tab>` does not work anymore, because
> the builtin completions are found first. We could overwrite
> BASH_COMPLETION_USER_DIR to have the highest precedence, but I think
> that most users use `./manage.py` anyways (it is what we recommend in
> our README) and the current approach seems to be the most canonical way
> to do this within nix packages.

How much do we care about completion for `python manage.py <Tab>` over `./manage.py <Tab>`? In my mind, the gain of supporting the former is less than the piece of mind of using the canonical packaging for the completion files and not messing with env variables ourselves.

Is anyone using the long form and would be annoyed by switching to `./manage.py`?